firestore/FirestoreSwiftUIExample/FirestoreSwiftUIExampleApp.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  FirestoreSwiftUIExampleApp.swift
+//  FirestoreSwiftUIExample
+//
+
+import SwiftUI
+import Firebase
+
+@main
+struct FirestoreSwiftUIExampleApp: App {
```

Comment:
I've been thinking about whether we should promote the more complete setup (including @ UIApplicationDelegateAdaptor).

@maksymmalyhin is working on a proposal to eliminate the need for method swizzling, and this would require developers to hook into the application delegate. See go/firebase-app-delegate-api-review for details.

So I think it might be useful to start socialising the idea of using @UIApplicationDelegateAdaptor.

What are people's thoughts - @morganchen12 @maksymmalyhin @racheldoshcollins ?

Reply:
That's unnecessary for Firestore, which doesn't have any swizzling-dependent behavior. I'm ok with omitting that step in quickstarts that don't need it, but we will definitely want the complete setup as you're describing in Analytics or Messaging.

Reply:
The existing UIKit version of this app uses Auth, however; I just haven't gotten around to including Auth in the SwiftUI version yet. Does Auth have swizzling-dependent behavior? In that case, the more complete setup may be the way to go, but we can delay updating it until I add Auth to this quickstart (which isn't a high priority).

Reply:
As far as I can tell, Auth doesn't use swizzling. At least I haven't run into any issues so far when using the lighter weight SwiftUI + Firebase setup.

Agree with Morgan that it's fine to go with the light-weight (i.e. init()-only) approach if we don't need a full AppDelegate.